src/main/java/com/nashtech/model/Vehicle.java

```diff
@@ -0,0 +1,50 @@
+package com.nashtech.vehicleapplication.model;
```

Comment:
no need to create the extra package, the `model` package is already there. Vehicle model is common for GCP as well as Azure